include/seqan/translation/translation.h

```diff
@@ -327,8 +327,8 @@ _translateImplLoopOMPWrapper(TTarget & target,
 {
     typedef typename Size<TTarget>::Type TPos;
     for (TPos i = 0; i < length(target); ++i)
-        _translateImplLoop(target, i, source, GeneticCode<CODE_SPEC>(),
-                           Frames_<frames>());
+        if (length(source[i/frames]) >= 3)
```

Comment:
Same here.
Could you also comment the if-condition. As it is not a complete trivial condition here.

Reply:
done